Explorer/Assets/Scripts/Utility/QuaternionUtils.cs

```diff
@@ -5,7 +5,7 @@ namespace Utility
 {
     public static class QuaternionUtils
     {
-        public const float DEFAULT_ERROR = 0.00001f;
+        public const float DEFAULT_ERROR = float.Epsilon;
```

Comment:
I made it higher on purpose, float.Epsilot is E-45, it's less than floating precision so it was always true even if the camera is not moved